src/monitors/AfterInstrMonitor.v3

```diff
@@ -0,0 +1,31 @@
+// Copyright 2023 Wizard Authors. All rights reserved.
+// See LICENSE for details of Apache 2.0 license.
+
+class AfterInstrMonitor extends Monitor {
```

Comment:
I think this name should be `PostInstructionTestTracer` or something similar. (and there should be a testcase for it), since it seems to mostly serve as a test case for the probe-after functionality.